Automattic/jetpack
# Update At A Glance Dashboard

Fixes # 4020

This updates the specs of the dashboard to match the new designs/information in p6TEKc-xF-p2. 

There are still a couple of kinks that need to be worked out in this, such as: 

- [x] more DRYness when showing the inactive/uninstalled state of the plugins
- [x] direct link from the SectionHeader `activate` and `install` links/buttons
- [x] The `loading` card for Backups is not 50%
- [x] Inactive stats module needs it's own new component (see screenshots in the p6TEKc-xF-p2)
- [x] `Active` text in the SectionHeaders for Pro cards need styling

**To test:** 

Non-pro: 
- activate/deactivate non-pro modules.  Make sure data displays correctly. 

Akismet: 
- Active and installed 
- Active and installed with no API key 
- Inactive and installed
- Uninstalled

Scan: 
- VaultPress Active and installed 
- VaultPress Active and installed with spoofing a threat.  You must hardcode the value in place of this `const threats = this.props.getScanThreats();` in `scan.jsx`
- VP inactive but installed
- VP uninstalled

Backups: 
- VaultPress Active and installed 
- VP inactive but installed
- VP uninstalled
